app/src/cli/package/run.rs

```diff
@@ -0,0 +1,40 @@
+use ambient_native_std::asset_cache::AssetCache;
+use clap::Parser;
+
+use crate::{cli::ClientCli, client};
+
+use super::{serve, HostCli, PackageArgs};
+
+#[derive(Parser, Clone, Debug)]
+/// Builds and runs the package locally
+pub struct Run {
```

Comment:
So the idea is for `run` to be under `package`? `ambient package run`?

Reply:
The idea was that I'd move all the commands there, and then have aliases back to them, as-is the case with campfire - so that you could always have a consistent interface, but also a convenient one. However, didn't happen, so it's still just `ambient run`.